### ISSUE TITLE
default_state.pkl problem

### ISSUE THREAD

**Issue by concu72** [9homme/awesome-bot#2]
Hello, first of all thank you for sharing your bot. After a few tweaks the bot works really great. There is a small problem that I noticed, which is not very important; When the bot is run, there is a file named default_state.pkl created by the bot in the directory with the files. If the bot is stopped and changes are made in user.cfg, it is necessary to manually delete the file named default_state.pkl that I mentioned, otherwise it does not see the changes in user.cfg. If I make changes in user.cfg, I manually delete the file I mentioned and run the bot, no problem.

   Also, the risk_reward_ratio part does not accept a decimal value. It is necessary to give an integer value in its current form. I think it would be much nicer if the risk_reward_ratio rate part could be given a decimal value.

   If you have a Discord, Telegram etc group regarding bot, I would like to join there.

   Thank you again for your efforts.

**Comment by concu72** [comment]
![1](https://user-images.githubusercontent.com/92368559/220102935-d2f766b3-38b5-4256-b7d2-d3d36f769611.JPG)

**Comment by concu72** [comment]
![3](https://user-images.githubusercontent.com/92368559/220103693-ea439e47-86a6-4efa-96ae-137171d059b8.JPG)

Examples of transactions made by the bot,

**Comment by concu72** [comment]
![2](https://user-images.githubusercontent.com/92368559/220104816-a25d5d36-dca9-4356-a111-93bab22be409.JPG)

Bot running

**Comment by 9homme** [comment]
@concu72 Hey, thank you so much for using the bot and sharing the feedback.
Will make a fix and let you know when it's done.

**Comment by concu72** [comment]
`C:\Users\Administrator\Desktop\awe>main.py
Loading config from [default]
Number of arguments: 1 arguments.
Argument List: ['C:\\Users\\Administrator\\Desktop\\awe\\main.py']
current state for AVAXUSDT datetime: 2023-02-22 00:30:00 position: long price: 21.54 quantity: 1.0 exit_price: None take_profit: None total_revenue: 7.61028
Analyzing AVAXUSDT last_close_price: 20.903 latest_price: 20.88 signal_type: wavetrend fast_signal: -62.89423917187632 slow_signal: -66.3125175831513
2023-02-22 05:39:00 AVAXUSDT wavetrend closed: 20.903 fast_signal: -62.89423917187632 slow_signal: -66.3125175831513
Some error occured but just skip, message: {'e': 'kline', 'E': 1677044475698, 's': 'AVAXUSDT', 'k': {'t': 1677044340000, 'T': 1677044519999, 's': 'AVAXUSDT', 'i': '3m', 'f': 212381121, 'L': 212381247, 'o': '20.89000000', 'c': '20.85000000', 'h': '20.92000000', 'l': '20.84000000', 'v': '2154.46000000', 'n': 127, 'x': False, 'q': '44975.47240000', 'V': '840.17000000', 'Q': '17540.01890000', 'B': '0'}}, error: '>' not supported between instances of 'float' and 'NoneType'
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\awe\main.py", line 44, in handle_socket_message
    strategy.process_closed_candle_for_current_position(message_datetime)
  File "C:\Users\Administrator\Desktop\awe\strategy.py", line 237, in process_closed_candle_for_current_position
    if new_exit_price > state.exit_price:
TypeError: '>' not supported between instances of 'float' and 'NoneType'`

This is the example of the problem I mentioned in this title. The bot last entered the AWAX trade. Since the bot stopped due to msg["k"]["t"] / 1000.0 problem error, after closing the position manually, when I run it again, because the wallet cannot see the position, it cannot detect the position that has been closed manually and tries to close it, and if such a position does not exist in the wallet, if the default_state If it is run without manually deleting the .pkl file, it stays like this. But as I wrote before, if the bot is run after deleting the default_state.pkl file before running, there is no such problem.
